appveyor.yml

```diff
@@ -0,0 +1,28 @@
+version: "{build}"
+
+shallow_clone: true
+
+environment:
+  matrix:
```

Comment:
Nit: can this be sorted latest version first like we do in .travis.yml?

Reply:
Yeah they have some aliases. I will add it. But how about always installing the latest version? For example, right now Node.js 12.4.0 is out but AppVeyor is still using 12.13.1

Reply:
What I meant is let's have appveyor.yml list "Current", then 12, then 10, then 8 in the matrix (newest first instead of oldest first).

As for which versions to run the goal is to always run the latest of each supported semver-major.  For nyc 15 this is node.js 8, 10, 12 and for now 13.

Reply:
Alright, I will switch these. BTW Travis is not testing Node.js 12 anymore since `node` means 13